code/src/components/QuestionThree.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+// import RadioOption from './RadioOption'
+
+
+
+const QuestionThree = ({ flower, setFlower, item }) => {
+
+    const onFlowerChange = (event) => {
+        setFlower(event.target.value)
+    }
+
+    return (
+        <>
+            <section>
+                <label key={item} htmlFor={item}>{item}</label>
+                    <input 
+                        name="flower" 
+                        id={flower} 
+                        type="radio" 
+                        value={item}
```

Comment:
All of the props could be the same e.g. all {flower}, and that includes the htmlFor. Then you just need flower={flower} in Form.js instead of both of them. And then also use flower instead of item in the map.